### PR TITLE
feat: added the companies screen

### PR DESCRIPTION
**Pull request by AJcreativity001**

### CODE REVIEW

File: src/Modules/Companies/components/card.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+const Cards = (): JSX.Element => {
+  const CardData = [1, 2, 3, 4, 5, 6];
+  return (
+    <Box
+      pt={{ base: "1rem", md: "2.5rem", lg: "3.188rem" }}
+      display="flex"
+      flexWrap="wrap"
+      gap={{ base: "1.5rem", md: "2rem", lg: "2rem" }}
+    >
+      {CardData.map((c, index) => {
```
Comment (copy-ninja1) [inline src/Modules/Companies/components/card.tsx:17]: this component should only hold a single card, do not loop it here. the loop should be on the Companies page

File: src/Modules/Companies/components/card.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        return (
+          <Box
+            w={{ base: "100%", md: "22.604rem", lg: "22.604rem" }}
+            bgColor="#ffffff"
+            px={{ base: ".8rem", md: ".875rem", lg: "0.875rem" }}
+            py={{ base: "1.5rem", md: "1.5rem", lg: "1.5rem" }}
+            borderRadius="0.75rem"
+            boxShadow="0px 4px 12px rgba(194, 194, 194, 0.2)"
```
Comment (copy-ninja1) [inline src/Modules/Companies/components/card.tsx:25]:
```suggestion
            boxShadow="100"
```

File: src/Modules/Companies/components/header.tsx [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+const CompaniesHeader = (): JSX.Element => {
+  return (
+    <Box
+      w="100%"
+      px={{ base: "1rem", md: "2rem", lg: "2.5rem" }}
+      h={{ base: "3.5rem", md: "3.5rem", lg: "3.438rem" }}
+      bgColor="#ffffff"
+      borderRadius="0.75rem"
+      boxShadow="0px 4px 12px rgba(194, 194, 194, 0.2)"
```
Comment (copy-ninja1) [inline src/Modules/Companies/components/header.tsx:12]:
```suggestion
      boxShadow="200"
```

**Review by copy-ninja1 — CHANGES_REQUESTED** [review]

**Comment by copy-ninja1** [comment]
please complete the company flow, on click of the company card a modal should come up showing you more details about the company
https://www.figma.com/file/VxoknphZySezGNvfr63r42/Afrisplash-Final-UI?node-id=9824-86098&t=d99B7E7SxTR84HXF-4

**Review by sammy-code98 — COMMENTED** [review]
great work  chief
just  fix the changes @copy-ninja1  pointed out

**Review by copy-ninja1 — APPROVED** [review]

**Pull request merged**